app/models/deploy.rb

```diff
@@ -110,7 +110,7 @@ def self.finished_naturally
   end
 
   def self.prior_to(deploy)
-    deploy.persisted? ? where("#{table_name}.id < ?", deploy.id) : self
+    deploy.persisted? ? where("#{table_name}.id < ?", deploy.id) : all
```

Comment:
To be fair, calling `#all` turns it into an array instead of an arel, which would interfere with chaining. So it's probably better to use `where('1=1')`, unless you know of another method that returns an arel but preserves chaining.